platform/core/src/utils/generateAcceptHeader.ts

```diff
@@ -17,6 +17,7 @@ const generateAcceptHeader = (
     acceptHeader.push('type=application/octet-stream');
   }
 
+  acceptHeader.push('transfer-syntax=*');
```

Comment:
This will add a second transfer-syntax=* header.  I think you need to set requestTransferSyntaxUID to '*' when it is undefined.